assets/js/blocks/featured-product/index.js

```diff
@@ -38,7 +38,16 @@ registerBlockType( 'woocommerce/featured-product', {
 	supports: {
 		align: [ 'wide', 'full' ],
 		html: false,
-		color: true,
+		color: {
+			__experimentalDuotone:
+				'.wc-block-featured-product__background-image',
+			background: false,
+			text: true,
+		},
+		spacing: {
+			padding: true,
+			__experimentalSkipSerialization: true,
```

Comment:
Given this is `__experimental` would we want the `spacing` to be wrapped in `isFeaturePluginBuild()` below?

Reply:
Looks like we should! Thanks for the catch, I'm still a bit confused with the whole `__experimental` thing.